src/lib/support/StringBuilder.h

```diff
@@ -50,6 +51,15 @@ class StringBuilderBase
         return Add(buff);
     }
 
+    /// Append a char value
+    StringBuilderBase & Add(char value)
+    {
+        char buff[2];
+        snprintf(buff, sizeof(buff), "%c", value);
```

Comment:
Lets implement this for real ... this is very odd. We want printable so how about:

```
uint8_t actual = std::is_print(value) ? static_cast<uint8_t>(value) : '.';
mBuffer.Put(actual);
NullTerminate();
return *this;
```